sql-odbc/build_mac_debug64.sh

```diff
@@ -2,13 +2,18 @@
 # $BITNESS=64
 
 cd src
-git clone -b "1.7.329" "https://github.com/aws/aws-sdk-cpp.git"
+vcpkg install
 cd ..
 
+vcpkg_installed_dir='x64-osx'
+if [[ $(uname -m) == 'arm64' ]]; then
```

Comment:
Can you use $OSTYPE or $MACHTYPE to determine OS version? Both are built-in bash variables.

Reply:
Testing a variable is simpler than running an executable, capturing its output and then checking it.

Looking at it now,  I think it has to check for `arm64-apple-darwin*` or similar to make sure the script is running on  macOS on ARM --`arm64` is too generic. Most of Linux seems to settle on `aarch64` to mean the same thing, FreBSD uses `arm64`.

Even just looking at `if [[ $(uname -m) == 'arm64' ]]; then` by itself it's not obvious it is intended to pass only on macOS without knowing that `uname -m` on Linux on ARM reports `aarch64` but on macOS -- `arm64`.